apps/web/.env.example

```
@@ -1,7 +1,8 @@
 DATABASE_URL="postgresql://postgres:password@localhost:5432/inboxzero?schema=public"
 DIRECT_URL="postgresql://postgres:password@localhost:5432/inboxzero?schema=public"
 
-NEXTAUTH_SECRET= # Generate a random secret here: https://generate-secret.vercel.app/32
+BETTER_AUTH_SECRET= # Generate a random secret here: https://generate-secret.vercel.app/32
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)